lib/notification/adapter/sqlite.js

```diff
@@ -0,0 +1,33 @@
+const { markdown2Html } = require('../../services/markdown');
+const Database = require('better-sqlite3');
+
+/**
+ * simply prints out the found data to the console
```

Comment:
please update the comment. This is a copy & paste from the console adapter ;)

Reply:
I'll do it myself. next time, please make sure to cleanup when copy and paste ;)  thanks